### ISSUE TITLE
[Android] Implementation of Material 3 toggles

### ISSUE THREAD

**Issue by meenbeese** [organicmaps/organicmaps#4674]
**Is your feature request related to a problem? Please describe.**

No

**Describe the solution you would like**

Since you are in the process of converting some of the splash screens from Material 2 to M3, I thought that it might be a good time to do the same thing for toggles.

I am talking specifically about the toggles in the settings menu which look old and outdated in Android; compared to the rest of the app which looks modern.

Maybe you would want to first make the toggles green and then convert it to M3, but either way it would be a nice improvement for sure.

**Describe alternatives you have considered**

None

**Additional context**

Google has nice guidelines if you need them:
https://m3.material.io/components/switch/overview

Thank you for your tireless contributions.

**Comment by n1snt** [comment]
Hey if no one is working on this I would love to have a try at this.

**Comment by biodranik** [comment]
@n1snt Please don't spam by comments like this one. If you want to fix something — do it without asking.

**Comment by n1snt** [comment]
## Updates:
I was able to add the toggles by changing the toggle style in theme-base.
I also switched the base theme to material 3 because the toggles don't appear properly without changing the base theme.

Light theme:
Settings             |  Main Screen
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/46300244/229895815-8172a50b-5fb6-4d9c-9b94-6baf76abbd23.png) | ![image](https://user-images.githubusercontent.com/46300244/229896559-ae3a665e-758d-4f1b-8006-43db7e37e53a.png) 

Dark theme
Settings             |  Main Screen
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/46300244/229897884-61ca4e4e-622c-46af-9c5a-eb849a9a400f.png) | ![image](https://user-images.githubusercontent.com/46300244/229897928-79111158-3cd1-4660-b464-452387f5c22e.png)


Dark mode needs a lot of tuning.

**Comment by Jean-BaptisteC** [comment]
Check this https://github.com/orgs/organicmaps/discussions/4676

**Comment by Jean-BaptisteC** [comment]
Yes dark mode color is not easy to see, can we use blue color for toggles like in actually in release version ?

**Comment by meenbeese** [comment]
Can you create a PR for your changes @n1snt ? Even a draft PR would be fine to help us propose improvements and resolve the issue faster.

**Comment by Jean-BaptisteC** [comment]
Need more work, migrate-only switches don't render looks good with the rest of UI.
I have remove as possible custom components to simplify future migration to Material 3
